plugins/binanceExchange_ws_test.go

```diff
@@ -48,3 +48,39 @@ func Test_binanceExchangeWs_GetTickerPrice(t *testing.T) {
 		return
 	}
 }
+
+func Test_binanceExchangeWs_GetOrderBook(t *testing.T) {
+	if testing.Short() {
```

Comment:
Can remove this testing.Short() check. Don’t think it should be needed for this function